csv-data-management/info.json

```diff
@@ -331,6 +331,124 @@
         }
       ],
       "open": true
+    },
+    {
+      "title": "Join and Extract Data from two CSV",
+      "operation": "join_two_csv_and_extract_data",
+      "description": "This action will extract data by join of two CSV file. User can select option to deduplicate rows based on columns",
```

Comment:
by join of two CSV file ---> by joining two CSV files.

Reply:
User can select option to deduplicate rows based on columns -> this is very ambiguous stmt. Please make it more user friendly

Reply:
addressed